src/components/NavItem/NavItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./NavItem.css";
+
+const NavItem = ({ text, location, className }) => (
+  <span className={"NavItem " + className}>{text}</span>
+);
+
+NavItem.propTypes = {
+  text: PropTypes.string.isRequired,
+  location: PropTypes.string
```

Comment:
In the future, we may want to use the eslint default props rule? -> https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md